lxd/instance/drivers/driver_lxc.go

```diff
@@ -5700,6 +5700,12 @@ func (d *lxc) MigrateSend(args instance.MigrateSendArgs) error {
 			}
 		}
 
+		postMigrateSendErr := d.postMigrateSendCommon(d, args.ClusterMoveSourceName)
```

Comment:
I think we should only run postMigrateSendCommin if the `err` is non-nil indicating the migration send completed successfully.